backend/app/routes/faq/faq.js

```diff
@@ -0,0 +1,39 @@
+const {authMiddleware, generateJWT} = require('../../../helpers/middlewares/auth');
+const express = require ("express");
+const FAQ = require('../../models/faq');
+const router = express.Router();
+const { ErrorHandler } = require('../../../helpers/error');
+const constants = require('../../../constants');
+
+module.exports = async (req, res, next) => {
+    try {        
+        const {
+            question,
+            answer,
+            isActive,
```

Comment:
you are trying to set fields which are not present in the request body, check that